style.css

```diff
@@ -34,6 +39,99 @@ a{
     height:10%;
 }
 
+/* Workout/Program displays */
+.program-name{
+    font-size:3vh ;
+    text-align:left;
+}
+
+/* Workout/Program displays inputs and buttons */
+.workout-input-box{ /* rep and weight boxes */
+    background-color:D9D9D9;
```

Comment:
I assume all these colors are working, but I'm surprised because you're missing the `#` before all the hex colors in this file. I would have thought you'd need that.